packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -1,3 +1,146 @@
+* The latest versions the [salesforcedx-vscode-core](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-core) and [salesforcedx-vscode-apex-debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger) extensions (we suggest that you install all extensions in the [salesforcedx-vscode](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) extension pack)
+
+## Set Up the Apex Debugger
+The first time that you use the Apex Debugger in VS Code, complete these setup steps.
+
+1. Add the `DebugApex` feature to a scratch org definition file:  
+  `"features": "DebugApex"`
+1. In VS Code, run **SFDX: Create a Default Scratch Org**. 
+1. Choose the scratch org definition file that includes the `DebugApex` feature.
+1. When your scratch org is ready, assign permissions for the Apex Debugger to the org’s admin user.  
+    1. In VS Code, run **SFDX: Open Default Scratch Org**.
+    1. In your browser, from Setup, enter `Permission Sets` in the Quick Find box, then select **Permission Sets**.
+    1. Create a permission set.
+    1. Give the permission set a name that you can remember, such as `Debug Apex`.
+    1. In the “Select the type of users who will use this permission set” section, choose **None** from the User License drop-down list. Choosing None lets you assign the permission set to more than one type of user.
+    1. Save your changes.
+    1. Click **System Permissions**.
+    1. Click **Edit**.
+    1. Enable **Debug Apex**. Enabling this permission enables the set of permissions that your user needs. 
+    1. Save your changes.
+    1. Click **Manage Assignments**.
+    1. Click **Add Assignments**.
+    1. Select the users to whom you want to assign the permission set, and then click **Assign**.
+1. **Optional**: In VS Code, run **SFDX: Pull Source from Default Scratch Org**. Then, add your new permission set to your source control repository. If you have a copy of the permission set in your Salesforce DX project, you can assign permissions to scratch org users by running `sfdx force:user:permset:assign -n Your_Perm_Set_Name`.
+1. In VS Code, create a launch configuration for the Apex Debugger.
+    1. To open the Debug view, in the VS Code Activity Bar, click the bug icon (hover text: Debug).
+    1. To open your `launch.json` file, click the gear icon (hover text: Open launch.json).
+    1. Within the `"configurations"` array, add a `"Launch Apex Debugger"` configuration:
+        ```
+        "configurations": [
+            {
+                "name": "Launch Apex Debugger",
+                "type": "apex",
+                "request": "launch",
+                "sfdxProject": "${workspaceRoot}"
+            }
+        ]
+        ```
+    1. Save your `launch.json` file.
+
+## Debug Your Code
+After you’ve set up the Apex Debugger, set breakpoints and start a debugging session. Then, debug your code.
```

Comment:
I had to read this first sentence a couple times. I know using the imperative is correct according to our style guide, it read like a series to me. After you set up, set blah, and start...(all related to 'after') - so consider rewriting, or adding some extra words. Nice job! You set up the Apex Debugger. Now, set breakpoints ...and blah

Reply:
Yeah, I actually thought the same thing but I went ahead and followed the style guideline anyway. But, I'll revise this. Thanks for the input.